build/cli/src/lib.rs

```diff
@@ -835,6 +837,12 @@ pub async fn main_internal(config: Option<enso_build::config::Config>) -> Result
                 .await?;
 
             prettier::check(&ctx.repo_root).await?;
+
+            let js_modules_root = ctx.repo_root.join("app/ide-desktop");
+
+            Npm.cmd()?.current_dir(&js_modules_root).args(["run", "typecheck"]).run_ok().await?;
+
+            Npx.cmd()?.current_dir(&js_modules_root).args(["eslint", "."]).run_ok().await?;
```

Comment:
Why are we creating these newlines here? I don't think they are needed - they just make the code longer.

Reply:
should be fixed now